src/psr4/Bridge.php

```diff
@@ -868,16 +873,18 @@ private function _check_assets()
                 ? $this->config->get( 'paths.base' )
                 : __DIR__;
             foreach ( $this->config->get( 'autoenqueue.assets' ) as $asset ) {
-                if ( $file->exists( assets_path( $asset['asset'], $dir ) ) )
+                if ( $file->exists( assets_path( $asset['asset'], $dir ) ) ) {
+                    $flag = $asset['flag'] ? $asset['flag'] : $asset['footer'];
                     $this->add_asset(
                         $asset['asset'],
                         array_key_exists( 'enqueue', $asset ) ? $asset['enqueue'] : true,
                         $asset['dep'],
-                        $asset['footer'],
+                        $flag,
```

Comment:
Please reduce memory allocation, we do not need to create the variable `$flag`. You can add this conditional directly when filling `add_asset()` parameter.

Also, please test auto-enqueue via config file, and make sure backward compatibility works. I know there are a lot of projects using the `footer` setting in the config file, this should not break compatibility. If this is tested, the it is fine.

Reply:
Thanks @amostajo, I've used the ternary conditional in place of the variable and updated it to account for when neither flag nor footer is set in the config. I'll make some time to get a test up later this week.

Reply:
Sure thing! Ping me when you have finished testing.

Reply:
Thanks for the merge @amostajo, sorry for my delay.